### PR TITLE
Add props and transforms configuration to anonymize ansible output splunk (OPS-3241)

### PR DESCRIPTION
**Pull request by nadeemshahzad**
…lunk

Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

**Comment by nadeemshahzad** [comment]
I tested this locally on docker, this will anonymized ansible-command and ansible-setup data in syslog.
splunk

**Comment by nadeemshahzad** [comment]
File "props.conf" is already on splunk server so I picked up from there and added anonymize stanza in file.
```
[syslog]
TRANSFORMS-anonymize = ansible-output
```

File: playbooks/roles/splunk-server/templates/opt/splunk/etc/system/local/transforms.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[ansible-output]
+REGEX = ansible-(.*)
+FORMAT = ansible-output#####################
```
Comment (feanil) [inline playbooks/roles/splunk-server/templates/opt/splunk/etc/system/local/transforms.conf.j2]: Update the output to be `ansible-output: Anonymized for security.  ########################`

Also add a comment above this line with the ticket and an explanation of why we're doing this.

**Review by feanil — APPROVED** [review]
A few small comments but otherwise I think this is good to go.

**Comment by feanil** [comment]
@coryleeio I seem to recall that when we last tried to run this playbook on the splunk machine we ran into some issues.  We lost some indexes or something?  @nadeemshahzad or @coryleeio do either of your recall this?  Has this been resolved or is there a workaround?

**Comment by nadeemshahzad** [comment]
@feanil I don't exactly remember this but I run the splunk play against server with check mode and diff is showing lot of changes. I have updated the ticket with Ansible dry run output.

**Pull request merged**